allennlp_models/pair_classification/dataset_readers/snli.py

```diff
@@ -45,6 +46,7 @@ def __init__(
         combine_input_fields: Optional[bool] = None,
         **kwargs,
     ) -> None:
+        kwargs["manual_distributed_sharding"] = True
         super().__init__(**kwargs)
```

Comment:
I think it would be more readable to call `super().__init__(manual_distributed_sharding=True, **kwargs)`.